pyproject.toml

```diff
@@ -0,0 +1,119 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=62", "setuptools_scm[toml]>=8.0"]
+
+[project]
+name = "tlo"
+authors = [
+    {name = "Thanzi La Onse Model Development Team", email="a.tamuri@ucl.ac.uk"},
```

Comment:
@tamuri I copied this across from previous `setup.py` but we could either remove `email` field if your prefer or use `arc.collaborations@ucl.ac.uk` for example?

Reply:
We can remove email field, thanks.